src/main/java/uk/gov/justice/digital/delius/controller/api/AppointmentController.java

```diff
@@ -35,6 +44,15 @@ public AppointmentController(OffenderService offenderService, AppointmentService
         this.appointmentService = appointmentService;
     }
 
+    @PreAuthorize("hasRole('ROLE_COMMUNITY_INTERVENTIONS_UPDATE')")
+    @RequestMapping(value = "/offenders/crn/{crn}/sentence/{sentenceId}/appointment", method = RequestMethod.POST, consumes = "application/json")
```

Comment:
This could be a POST to '.../appointments' to create the item in the collection

Reply:
This has been done